music_assistant/providers/fanarttv/__init__.py

```diff
@@ -140,27 +140,27 @@ async def get_album_metadata(self, album: Album) -> MediaItemMetadata | None:
         self.logger.debug("Fetching metadata for Album %s on Fanart.tv", album.name)
         if data := await self._get_data(f"music/albums/{mbid}"):
             if data and data.get("albums"):
-                data = data["albums"][mbid]
-                metadata = MediaItemMetadata()
-                metadata.images = []
-                for key, img_type in IMG_MAPPING.items():
-                    items = data.get(key)
-                    if not items:
-                        continue
-                    for item in items:
-                        metadata.images.append(
-                            MediaItemImage(
-                                type=img_type,
-                                path=item["url"],
-                                provider=self.domain,
-                                remotely_accessible=True,
+                if album := data["albums"][mbid]:
+                    metadata = MediaItemMetadata()
+                    metadata.images = UniqueList()
```

Comment:
I just added a small helper into the models package so we dont have to instantiate the UniqueList everywhere:

https://github.com/music-assistant/models/commit/ce2055e82cda7dd91bb4379d22030fbf22e0f057

Reply:
Nice. I can bump the tag in this PR and use it if you do a release.

Reply:
models 1.1.16 is released with this change